src/status_im2/contexts/communities/overview/view.cljs

```diff
@@ -273,12 +273,15 @@
             :last-item-style style/last-community-tag
             :container-style style/community-tag-container}])
         [join-community community pending?]]
+       
+       (when (or (and token-permissions joined)
+               (not token-permissions))
        [channel-list-component
```

Comment:
Indentation looks a bit weird here. I guess a format will fix it.

Reply:
tks, I will take a look also a good opportunity to fix again my auto format!

Reply:
fixed =)